platform/migration/platform-migratable-util/pom.xml

```diff
@@ -90,18 +90,18 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
```

Comment:
I changed them since we will most likely change the code and test cases as we refine the export/import process.  As a result, we will probably end up with different coverage ratios at the end.